.github/workflows/govcloud.yml

```diff
@@ -16,172 +12,20 @@ on:
 jobs:
-  deploy-forms-to-paas:
-    environment: ${{ github.event.inputs.environment }}
-    runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      packages: write
-    env:
-      DOCKER_IMAGE: communitiesuk/digital-form-builder/runner
-    steps:
-      - name: Checkout repo
-        uses: actions/checkout@v2
-        with:
-          ref: ${{ github.event.inputs.branch }}
-
-      - name: Set environment variables
-        run: |
-          echo "DOCKER_IMAGE_TAG=${{ env.REGISTRY }}/${{ env.DOCKER_IMAGE }}:$GITHUB_SHA" >> $GITHUB_ENV
-
-      - name: list environment
-        run: |
-          echo $GITHUB_ENV
-
-      - name: Log in to the Container registry
-        uses: docker/login-action@v1
-        with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Build and push docker image
-        uses: docker/build-push-action@v2
-        with:
-          context: .
-          tags: |
-            ${{ env.DOCKER_IMAGE_TAG }}
-          push: true
-          file: Dockerfile
-          build-args: |
-            COMMIT_SHA=$GITHUB_SHA
-
-      - name: Install cloudfoundry
-        shell: bash
-        id: install-cf-cli
-        run: |
-          wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
-          echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
-          sudo apt-get update
-          sudo apt-get install cf8-cli
-
-      - name: Deploy to Gov.uk PaaS Dev
-        id: deploy-to-paas-dev
-        shell: bash
-        run: |
-          cf api ${{ secrets.CF_API }}
-          cf auth "${{ secrets.CF_USER }}" "${{ secrets.CF_PASSWORD }}"
-          cf target -o dluhc-funding-service-design -s sandbox
-          CF_DOCKER_PASSWORD=${{ secrets.GHCR_PAT }} cf push funding-service-design-form-runner-dev -f ./manifest.yml \
-            --docker-image ${{ env.DOCKER_IMAGE_TAG }} \
-            --docker-username ${{ github.actor }} \
-            --var RSA256_PUBLIC_KEY_BASE64="${{secrets.RSA256_PUBLIC_KEY_BASE64}}"
-
-      - name: Deploy to Gov.uk PaaS test
-        id: deploy-to-paas-test
-        shell: bash
-        run: |
-          cf api ${{ secrets.CF_API }}
-          cf auth "${{ secrets.CF_USER }}" "${{ secrets.CF_PASSWORD }}"
-          cf target -o dluhc-funding-service-design -s sandbox
-          CF_DOCKER_PASSWORD=${{ secrets.GHCR_PAT }} cf push funding-service-design-form-runner-test -f ./manifest.yml \
-            --docker-image ${{ env.DOCKER_IMAGE_TAG }} \
-            --docker-username ${{ github.actor }} \
-            --var RSA256_PUBLIC_KEY_BASE64="${{secrets.RSA256_PUBLIC_KEY_BASE64}}"
-
-
-  # TODO: Add back when form runner is switched to use fork (ZAP form 404 issue is resolved here)
```

Comment:
can we add this back in now 👀 ?

Reply:
Yes - I've put it in the other workflow now tho as it targets the forms https://github.com/communitiesuk/funding-service-design-frontend/pull/92/commits/313a54e3641d225939fe1a071a4615c3c86f9f16

Reply:
Disabled again as it doesn't like basic auth and since that's only temporary I didn\t want to spend time making it work. Will reinstate it when the domain stuff is all sorted